test/publish/basic_test.rb

```diff
@@ -0,0 +1,28 @@
+require 'browser_helper'
+
+describe 'publish-basic', :js do
+
+  it 'gets redirected to the login page' do
+    visit CONFIG.PUBLISH_HOST
+    page.must_have_content "You must login"
+  end
+
+  it 'can login' do
+    publish_login!
+    visit CONFIG.PUBLISH_HOST
+    page.must_have_content 'Home'
+    page.must_have_content 'Your Series'
+    page.must_have_content 'Your Standalone Episodes'
+  end
+
+  # TODO: Intl errors prevent waiting for spinners to disappear
+  # it 'can navigate to a story' do
+  #   publish_visit CONFIG.PUBLISH_HOST
+  #   all('publish-home-story')[1].click
+  #   publish_wait
+  #   page.must_have_content 'Edit Episode'
```

Comment:
Yeah, I'm sort of on the fence about how we'll proceed from here.  (Very specific test-user fixtures, vs creating everything you'll need).

I kind of think, since this is in staging, we could just have the user create their own fixtures as part of the tests.  (And hopefully also clean them up after).